ibis/backends/dask/execution/window.py

```diff
@@ -0,0 +1,86 @@
+"""Code for computing window functions in the dask backend."""
+
+from typing import Any, Optional, Union
+
+import dask.dataframe as dd
+
+import ibis.expr.operations as ops
+import ibis.expr.window as win
+from ibis.expr.scope import Scope
+from ibis.expr.typing import TimeContext
+
+from ..core import execute_with_scope
+from ..dispatch import execute_node
+from .util import (
+    _pandas_dtype_from_dd_scalar,
+    _wrap_dd_scalar,
+    add_partitioned_sorted_column,
+    make_meta_series,
+)
+
+
+def _post_process_empty(
+    result: Any,
+    parent: Union[dd.Series, dd.DataFrame],
+    timecontext: Optional[TimeContext],
+) -> dd.Series:
+    """Post process non grouped, non ordered windows.
+
+    dd.Series/dd.DataFrame objects are passed through, otherwise we conform
+    the output to the parent input (i.e. so the shape an partitioning matches).
+
+    dd.core.Scalar needs special handling so downstream functions can work
+    with it.
+    """
+    if isinstance(result, (dd.Series, dd.DataFrame)):
+        return result
+    elif isinstance(result, dd.core.Scalar):
+        # TODO this should be refactored with similar logic in util.py
+        # both solve the generalish problem we have of wrapping a
+        # dd.core.Scalar into something dask can work with downstream
+        # TODO computation
+        lens = parent.index.map_partitions(len).compute().values
+        out_dtype = _pandas_dtype_from_dd_scalar(result)
+        meta = make_meta_series(dtype=out_dtype)
+        delayeds = [_wrap_dd_scalar(result, None, out_len) for out_len in lens]
+        series = dd.from_delayed(delayeds, meta=meta,)
+        series, _ = add_partitioned_sorted_column(series, set_index=True)
+        return series[0]
+
+    else:
+        return parent.apply(lambda row: result, meta=(None, 'object'))
```

Comment:
What is this branch here?

Reply:
If you have a scalar that isn't delayed we can just insert it into every row of the parent.

Reply:
I see, can you comment here? (Better if you can validate/assert)

Reply:
I'll add a comment. 

Validate which part?

Reply:
Sorry, I was thinking to validate the object falling into the else branch is just a python scalar but I realized there probably isn't a very easy way to do that. So I am fine with just a comment XD

Reply:
> else branch is just a python scalar

Importantly this might not be a python scalar, it could be a numpy.ndarry. But we know at this point in the branching that we can use `apply` since the result isn't a dask object.

Reply:
I see. Fine to leave as well. It seems that the objects are can be anything that is not distributed Dask object